Sources/MustacheFoundation/Classes/Extensions/NSAttributedString+Extensions.swift

```diff
@@ -0,0 +1,43 @@
+
+// Credit https://stackoverflow.com/a/50064887/1366083
+
+import Foundation
+
+public extension NSMutableAttributedString {
+    
+    func set(linkString: String) {
```

Comment:
The api and/or naming seems a bit weird. What is the intended use case? To scan and style an existing AttributedString or add/append a string with embedded links. It could instead be an init or an append func.